src/user.rs

```diff
@@ -72,8 +72,15 @@ impl User {
 			bail!("missing `{}` values for `{}`", attribute, entry.dn)
 		}
 
-		let enabled = read_entry(&entry, &config.ldap.attributes.status)?
-			!= StringOrBytes::String(config.ldap.attributes.disable_value.clone());
+		let status_as_int = match read_entry(&entry, &config.ldap.attributes.status)? {
+			StringOrBytes::String(status) => status.parse::<i32>()?,
+			StringOrBytes::Bytes(status) => i32::from_be_bytes(
+				status.try_into().map_err(|_| anyhow!("failed to convert to i32 flag"))?,
+			),
+		};
+		let enabled =
+			!config.ldap.attributes.disable_bitflags.iter().any(|flag| status_as_int & flag > 0);
```

Comment:
I like this, though I suppose we may need to have more configuration options if we ever run into an LDAP schema which doesn't use integer values for this.